tensorflow.patch

```diff
@@ -139,8 +143,20 @@ index 0000000000..ff7b11cf1d
 +}  // namespace xla
 +
 +#endif  // TENSORFLOW_COMPILER_XLA_RPC_COMPUTATION_CLIENT_H_
+diff --git a/tensorflow/compiler/xla/rpc/grpc_service_main.cc b/tensorflow/compiler/xla/rpc/grpc_service_main.cc
+index c68c857..7110c70 100644
+--- a/tensorflow/compiler/xla/rpc/grpc_service_main.cc
++++ b/tensorflow/compiler/xla/rpc/grpc_service_main.cc
+@@ -48,6 +48,7 @@ int RealMain(int argc, char** argv) {
+ 
+   builder.AddListeningPort(server_address, ::grpc::InsecureServerCredentials());
+   builder.RegisterService(service.get());
++  builder.SetMaxReceiveMessageSize(INT_MAX);
```

Comment:
Is this still needed?

Reply:
Yes, otherwise server will complain the message is too large.